mllm/core/aops/ParallelLinearOp.hpp

```
@@ -0,0 +1,47 @@
+class ParallelLinearOp : public BaseOp {
+ public:
+  explicit ParallelLinearOp(const ParallelLinearOpOptions& options);
+
+  void load(const ParameterFile::ptr_t& ploader) override;
+  void trace(void* trace_context, const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+  void forward(const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+  void reshape(const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+  void setup(const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+  ParameterFile::ptr_t getParams() override;
+
+  inline const ParallelLinearOpOptions& options() const { return options_; }
```

Clang (14.0.6)

[error] function 'options' should be marked [[nodiscard]]

(modernize-use-nodiscard,-warnings-as-errors)